free-tunes/ExploreMusic/Repository/TracksModel.swift

```diff
@@ -22,16 +22,21 @@ struct Tracks: Codable {
 }
 
 struct Track: Codable {
-    let title: String
-    let artist: Artist
-    let album: Album
-    let trackLink: String
+    private let trackTitle: String
```

Comment:
These don't need to be `private` 😄

Reply:
I was trying to get the data protection on the model to prevent the data from changing in any way. unless if that's bad practice